semcore/date-picker/src/components/Calendar.jsx

```diff
@@ -184,14 +188,74 @@ class CalendarAbstract extends Component {
 
     fire(this, 'onHighlightedChange', highlighted.length ? [highlighted[0]] : []);
   };
+
+  getUnitProps({ date }) {
+    const { unitRefs } = this.asProps;
+    return {
+      ref: (node) => {
+        if (!date) return;
+        unitRefs[date] = node;
+      },
+    };
+  }
+
+  formatter = formatDDMMYY;
+  describeValue() {
+    const { locale, getI18nText, actionsDescribing } = this.asProps;
+    const value = this.asProps.value || [];
+    const compare = this.asProps.compare || [];
+
+    if (!actionsDescribing) return null;
+
+    const t = (key, date) =>
+      getI18nText(key, date ? { date: this.formatter(date, locale) } : undefined);
+
+    let description = '';
+    if (value.length === 1 || compare.length === 1) {
+      description += t('selectingStarted') + '. ';
+    } else if (value.length === 2 || compare.length === 2) {
+      description += t('selectingFinished') + '. ';
+    }
+
+    if (actionsDescribing === 'range') {
+      if (value[0] || value[1]) description += t('dateRange') + ' ';
+      if (value[0]) description += t('fromDate', value[0]) + ', ';
+      if (value[1]) description += t('toDate', value[1]) + '. ';
```

Comment:
it will be a little bit strange description, if we haven't value[1]... no?

Reply:
It goes good in case if person in process of selecting, after clicking first calendar unit something like "selecting date range from x"

Can't imagine other cases when `value[1]` not available